app/src/main/java/io/appium/uiautomator2/utils/Attribute.java

```diff
@@ -42,6 +42,7 @@ public enum Attribute {
     SELECTED(new String[]{"selected"}),
     TEXT(new String[]{"text", "name"}),
     HINT(new String[]{"hint"}),
+    EXTRAS(new String[]{"extras"}),
```

Comment:
I think it would make sense to put the presence of this attribute in the XML under a setting. Otherwise it might grow too huge

Reply:
good idea

Reply:
added `includeExtrasInPageSource` to include it in the page source.